test/runtests.jl

```diff
@@ -29,8 +29,10 @@ const TRIXI_MPI_NPROCS = 3
   end
 
   @time if TRIXI_TEST == "all" || TRIXI_TEST == "parallel_2d"
+    # Based on `runtests.jl` from `MPI.jl` and `PencilArrays.jl`
+    # Precompilation disabled to prevent race conditions when loading packages
     mpiexec() do cmd
-      run(`$cmd -n $TRIXI_MPI_NPROCS $(Base.julia_cmd()) test_examples_parallel_2d.jl`)
+      run(`$cmd -n $TRIXI_MPI_NPROCS $(Base.julia_cmd()) --compiled-modules=no test_examples_parallel_2d.jl`)
```

Comment:
You could also use [PkgLock.jl](https://github.com/simonbyrne/PkgLock.jl) to precompile all packages without race conditions and use them afterwards in parallel.

Xref: https://discourse.julialang.org/t/precompilation-error-using-hpc/17094

Reply:
Thanks for the recommendation! Right now I am first trying to figure out why my tests stall without output:
https://travis-ci.com/github/trixi-framework/Trixi.jl/jobs/394696540#L363-L373

Reply:
Tracked in #214.